cocotb/runner.py

```diff
@@ -314,32 +389,27 @@ def get_abs_paths(paths: Sequence[PathLike]) -> List[str]:
 
     paths_abs: List[str] = []
     for path in paths:
-        if os.path.isabs(path):
-            paths_abs.append(os.path.abspath(path))
+        if PurePath(path).is_absolute():
+            paths_abs.append(str(Path(path).resolve()))
         else:
-            paths_abs.append(os.path.abspath(os.path.join(os.getcwd(), path)))
+            paths_abs.append(str((Path(os.getcwd()) / path).resolve()))
 
     return paths_abs
 
 
 class Icarus(Simulator):
+    def __init__(self) -> None:
+        super().__init__()
+        self.supported_languages = ("verilog",)
```

Comment:
Not sure about this but would this be better in this case?
```suggestion
    supported_languages = ("verilog",)
```

Reply:
It never changes.